packages/block-template/templates/html/block-metadata.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "html-block",
+  "version": "0.0.0",
+  "description": "Block Protocol block template for an HTML element-based block",
+  "author": {
+    "name": "HASH",
+    "url": "https://hash.ai"
+  },
+  "license": "MIT",
+  "schema": "block-schema.json",
+  "variants": [],
+  "blockType": {
+    "kind": "html"
```

I changed this (has been fixed in other templates)
```suggestion
    "blockType": "html"
```